docs/core/porting/cpp-cli.md

```diff
@@ -1,13 +1,16 @@
 ---
-title: Migrating C++/CLI projects to .NET Core and .NET 5+
-description: Learn about porting C++/CLI projects to .NET Core and .NET 5 and later versions.
+title: Migrating C++/CLI projects to .NET Core and .NET
+description: Learn about porting C++/CLI projects to .NET Core and the latest .NET versions.
 author: mjrousos
 ms.date: 03/29/2022
 ---
 
-# How to port a C++/CLI project to .NET Core or .NET 5
+# How to port a C++/CLI project to .NET Core or the latest .NET
 
-Beginning with .NET Core 3.1 and Visual Studio 2019, [C++/CLI projects](/cpp/dotnet/dotnet-programming-with-cpp-cli-visual-cpp) can target .NET Core. This support makes it possible to port Windows desktop applications with C++/CLI interop layers to .NET Core/.NET 5+. This article describes how to port C++/CLI projects from .NET Framework to .NET Core 3.1.
+Beginning with .NET Core 3.1 and Visual Studio 2019, [C++/CLI projects](/cpp/dotnet/dotnet-programming-with-cpp-cli-visual-cpp) can target .NET Core. This support makes it possible to port Windows desktop applications with C++/CLI interop layers to .NET Core/.NET. This article describes how to port C++/CLI projects from .NET Framework to the latest .NET.
+
+> [!NOTE]
+> C++/CLI is a fully supported language, however, we do not expect any future improvements. For example, we do not expect to add support for new .NET features like `Span<T>` or for new C++ standard versions. C++/CLI users should consider if this level of support is sufficient or if transitioning to ISO C++ and another .NET language, for example, C#, is a better approach for their needs. Migration approaches can be found [here](./cpp-cli-alts.md).
```

Comment:
> or for new C++ standard versions.

@tgani-msft should this instead be clarified to mean "no functional support" from newer C++ standard versions? I recall some mention of import support for modules but not export. Threading the needle on phrasing here is proving difficult.

Reply:
> or if transitioning to ISO C++ and another .NET language, for example, C#, is a better approach

I think this phrasing is a bit confusing, for two reasons:

- It says "another .NET language", but it is ambiguous which one was the other one. A naive read would think it is ISO C++ but you mean C++/CLI.
- I think you should simply say C# and not worry about the few folks that will pick another language instead. Also, your language is already sufficiently loose. It is not prescriptive.

Here's an actual suggestion.

```suggestion
> C++/CLI is a fully supported language, however, we do not expect any future improvements. For example, we do not expect to add support for new .NET features like `Span<T>` or for new C++ standard versions. C++/CLI users should consider if this level of support is sufficient or if transitioning to ISO C++ and C# is a better approach for their needs. Migration approaches can be found [here](./cpp-cli-alts.md).
```

Reply:
I agree. Thanks.